### PR TITLE
Bug 1171490 - Icons width not calculated correctly r=alive

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3f4b24ec5add1a8c74e9c9c415d84533b173e7f1

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=319ed2ba942ed77eba120ef9b72bddb9ef0b4e3d

File: apps/system/js/statusbar.js [hunk 1]
```diff
@@ -315,9 +315,16 @@
...
+      var appsWithoutRocketbar = ['isHomescreen', 'isAttentionWindow', 'isLockscreen'];
+      var noRocketbar = appsWithoutRocketbar.some(function(name) {
+        return !!(app[name]);
+      });
+
+      var chromeMaximized = !!(app.appChrome && app.appChrome.isMaximized());
+      var shouldMaximize = noRocketbar || chromeMaximized;
+
+      // Important: we need a boolean to make the toggle method takes the right decision
+      this.element.classList.toggle('maximized',  shouldMaximize || false);
```
Comment (alivedise) [inline apps/system/js/statusbar.js]: \O/

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d426d5f746a7934b89d6272f2ab3c350b4232e8a

**Pull request merged**